jsk_fetch_robot/jsk_fetch_startup/robots/jsk_fetch.urdf.xacro

```diff
@@ -37,4 +38,32 @@
     <child link="base_camera_mount" />
     <axis xyz="0 0 1" />
   </joint>
+
+  <joint name="head_box_virtual_joint" type="fixed">
+    <origin rpy="0 0 0" xyz="0 0 0" />
+    <parent link="head_pan_link" />
+    <child link="head_box_link" />
+  </joint>
+  <link name="head_box_link" >
+    <inertial>
+      <origin rpy="0 0 0" xyz="0 0 0" />
+      <mass value="1" />
+      <inertial ixx="1" ixy="0" ixz="0" iyy="1" iyz="0" izz="1" />
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0" />
+      <geometry>
+        <mesh filename="package://jsk_fetch_accessories/head_box/fetch_head_box_simplified.stl" scale="0.001 0.001 0.001" />
```

Comment:
i can't find this stl file, but where is this stl file?

Reply:
I'll add the stl with another PR.

Reply:
Please refer https://github.com/jsk-ros-pkg/jsk_robot/pull/1249

Reply:
Please refer to https://github.com/jsk-ros-pkg/jsk_robot/pull/1250